test.sh

```diff
@@ -24,6 +24,7 @@ export GO111MODULE=on
 
 function test_init_project {
   header_text "performing init project"
+  go mod init kubebuilder.io/test
   kubebuilder init --domain example.com <<< "n"
```

Comment:
we could also use the --repo in this scenario. However, it is fine as well.

Reply:
Didn't remember the `--repo` flag.